cmd/kubectl.go

```diff
@@ -5,8 +5,19 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"path"
+	"strings"
 
+	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
+	"gopkg.in/yaml.v2"
+
+	"github.com/rancher/norman/clientbase"
+	client "github.com/rancher/types/client/management/v3"
+)
+
+const (
+	kubeconfigpath = ".rancher/.cache/kubeconfig"
```

Comment:
Is a relative path always going to work?

Reply:
changed to use HOME env var